src/Avalonia.Controls.ColorPicker/Themes/Fluent/Fluent.xaml

```diff
@@ -28,6 +28,14 @@
         </VisualBrush.Visual>
       </VisualBrush>
 
+      <SolidColorBrush x:Key="ColorControlLightSelectorBrush" Color="#FFFFFFFF" />
+      <SolidColorBrush x:Key="ColorControlDarkSelectorBrush" Color="#E4000000" />
+      <DynamicResource x:Key="ColorControlDefaultSelectorBrush" ResourceKey="SystemControlForegroundBaseHighBrush" />
```

Comment:
@maxkatz6 Is DynamicResource expected to work like this in Avalonia? It fails at runtime:

`{"Unable to cast object of type 'Avalonia.Markup.Xaml.MarkupExtensions.DynamicResourceExtension' to type 'Avalonia.Media.IBrush'."}`

 If not I will have to get rid of these resources I guess. It doesn't work otherwise with light/dark theme changes.

Reply:
Yes, it wouldn't work. 
You can either keep it with static resource (and with broken theme switching) or use style directly. 
Just add a comment, so it can be revisited after theme dictionaries added.

Reply:
Thanks for confirming. I'll comment out these resources and use what they point to directly in the control themes.

If theme dictionaries allow us to "plug-in" to the light and dark resource dictionaries that will be great.

Reply:
You can check current implementation. Only one problem stopping it from merging - perf impact. It can be minimized if we would merge all resource dictionaries together, which hasn't been yet done.
https://github.com/AvaloniaUI/Avalonia/pull/8166

Reply:
IIRC in UWP they did merge all control resources into a single generic.xaml as well

Reply:
Yes, both WPF and UWP have a build script to merge all styles into one generic.xaml file. As long as Avalonia does this as part of the build and not in the actual source files I think it would make sense.